src/002_installation.md

```diff
@@ -78,25 +78,68 @@ Continue as described in Section [Running Tamarin](#sec:running-tamarin) to run
 Mac OS X {#sec:macosx}
 --------
 
-### Installing the pre-built Tamarin binary {#sec:MacOSBinInstall}
+### Automatic Installation of the Tamarin binary {#sec:MacOSBinInstall}
 
-To run Tamarin on Mac OS X you need to have Maude 2.7 and GraphViz. If you have [HomeBrew](http://brew.sh) installed, you can simply 
-`brew install homebrew/science/maude graphviz` to satisfy the pre-requisites of running the binary. Then follow from step 3 onwards.
+The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/) [Cask](https://caskroom.github.io/).
 
-Otherwise, to install Maude and GraphViz manually:
+If you already have this installed, it is as simple as running the following in your terminal:
+
+  * `brew cask install tamarin-prover`
+
+You can now run Tamarin from the command line by typing `tamarin-prover`. 
+Continue as described in Section [Running Tamarin](#sec:running-tamarin) to
+run Tamarin for the first time. (Warning: the current stable version of Tamarin
+v1.0.0 does not realise that Maude v2.7.1 is more than sufficient; it will
+currently give you a warning when you run `tamarin-prover test` but it is safe
+to ignore this.)
+
+#### Installing Homebrew & Cask
+If you don't have Homebrew and/or Cask installed, (which is highly recommended
+more generally): 
+
+1. Install [Homebrew](http://brew.sh/), by running this from the command line
+(no line-break): 
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
Probably not a good idea to copy-paste this; instead just link to brew.sh for install instructions. Otherwise they'll change it and we'll be out of date.